### PR TITLE
Update collect-logs to include Microsoft SQL example

### PR DESCRIPTION
**Pull request by itsthepo**
## Purpose of this pull request

This pull request... 

## Select the type of change

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 1]
```diff
@@ -527,6 +528,46 @@ On the first query run, the parameter represented with the dollar one `$1` will
 
 The last used tracking value will be stored persistently using the `file_storage` extension, allowing the collector to pick up where it left off between collector restarts.
 
+### Collecting logs from a mssql database
```
Comment (jpipkin1) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:531]:
```suggestion
### Collecting logs from an MSSQL database
```

**Review by jpipkin1 — APPROVED** [review]
Approved, @itsthepo. I made one minor update. I changed "mssql" to "MSSQL".

**Comment by jpipkin1** [comment]
@itsthepo, if you remove "Open Source Collection Team" as a reviewer, I can go ahead and merge the PR.

**Comment by itsthepo** [comment]
@jpipkin1 I can't seem to remove them

**Comment by jpipkin1** [comment]
@itsthepo, I'll go ahead and push through the merge without review from the "Open Source Collection Team".

**Pull request merged**